.travis.yml

```diff
@@ -5,6 +5,16 @@ before_install:
 - docker build . --build-arg RAILS_ENV=test -t app/test
 jobs:
   include:
+  - stage: Terraform
+    name: validate and lint the Terraform
+    script: |
+      git clone https://github.com/tfutils/tfenv.git ~/.tfenv
+      export PATH="$HOME/.tfenv/bin:$PATH"
+      tfenv install
+      bash -c "$(curl -fsSL https://raw.github.com/cloudfoundry-community/terraform-provider-cf/master/bin/install.sh)"
+      terraform init terraform
+      terraform fmt  -diff -check -recursive terraform
```

Comment:
A good idea to check this. I wonder about the ordering of this new step considering the number of future builds that will include Terraform changes compared to the much larger amount of Rails changes. As I think it runs jobs top to bottom I think we could move this until after the application tests have been run so that the more common build process gets faster feedback?

Reply:
Its a parallel job. So they run at the same time assuming there is enough capacity. It could be a lot a lot quicker but it has to wait for the docker image to be built.

Reply:
I've shifted this since it was easy enough